lib/modules/asset/AssetService.ts

```diff
@@ -52,6 +68,15 @@ export class AssetService {
     this.context = plugin.context;
     this.config = plugin.config;
     this.assetHistoryService = assetHistoryService;
+
+    this.registerAskEvents();
+  }
+
+  registerAskEvents() {
+    onAsk<AskAssetRefreshModel>(
+      "ask:device-manager:asset:refresh-model",
+      this.refreshModel
```

Comment:
The context is not bound for `refreshModel` (`this` will not be available inside the method)